docs/content/examples/tutorials/crowdsec.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Tutorials | Crowdsec'
+---
+
+!!! quote "What is Crowdsec ?"
+    
+    Crowdsec is an open source software that detects and blocks attackers using log analysis.
+    It has access to a global community-wide IP reputation database.
+
+    [Source][crowdsec]
+
+    [crowdsec]: https://www.crowdsec.net
+
+## Installation
+
+Crowdsec supports multiple [installation methods][crowdsec-installation-docs], however this page will use the docker installation.
+
+
+### Docker mailserver
+
+In the docker-compose definition of the docker-mailserver, create a bind volume containing the logs.
+
+```yaml
+services:
+  mailserver:
+    image: ghcr.io/docker-mailserver/docker-mailserver:latest
+    container_name: mailserver
+    # Provide the FQDN of your mail server here (Your DNS MX record should point to this value)
+    hostname: mail.example.com
+    ports:
+      - "25:25"
+      - "465:465"
+      - "587:587"
+      - "993:993"
+    volumes:
+      - /docker-data/dms/mail-data/:/var/mail/
+      - /docker-data/dms/mail-state/:/var/mail-state/
+      - /docker-data/dms/mail-logs/:/var/log/mail/
+      - /docker-data/dms/config/:/tmp/docker-mailserver/
+      - /etc/localtime:/etc/localtime:ro
+    environment:
+      - ENABLE_RSPAMD=1
+      - ENABLE_CLAMAV=1
+      - ENABLE_FAIL2BAN=0
+    restart: always
+```
```

Comment:
I think you can minimize noise in the yaml example.

It will vary per user, so it's simpler to just focus on the addition you want to add while communicating the config path from the top-level key `services`.

````suggestion
In your `compose.yaml` for the DMS service, add a bind mount volume for `/var/log/mail`. This is to share the DMS logs to a separate crowdsec container.

!!! example

    ```yaml
    services:
      mailserver:
        volumes:
          - /docker-data/dms/mail-logs/:/var/log/mail/
    ```
````